modules/chronograf-server/main.tf

```diff
@@ -0,0 +1,59 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# CREATE AN EC2 INSTANCE TO RUN CHRONOGRAF
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "aws_instance" "chronograf_server" {
+  ami                    = "${var.ami_id}"
+  instance_type          = "${var.instance_type}"
```

Comment:
You'll also need to: 

1. Set `iam_instance_profile`
1. Create the instance profile and attach an IAM role to it
1. Output the IAM Role ID
1. Set the `subnet_id` to a subnet passed in by the user
1. Expose `tenancy` with a reasonable default
1. Expose `ebs_optimized`  with a reasonable default
1. Expose `volume_type`, `volume_size`, and `delete_on_termination` in `root_block_device` with reasonable defaults